zcash_client_backend/CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this library adheres to Rust's notion of
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+### Changed
+- The api of `welding_rig::ScanningKey` has changed to accommodate batch
+  decryption and to correctly handle scanning with the internal (change) keys
+  derived from ZIP 316 UFVKs and UIVKs.
+- `welding_rig::scan_block` now uses batching for trial-decryption of 
+  transaction outputs.
+
+## [0.6.0] - 2022-05-11
+
```

Comment:
This also seems to have slipped in; we have not cut a `zcash_client_backend 0.6.0`.

```suggestion
```

The `Changed` section should be merged into the one below.